webapp/status_server.py

```diff
@@ -115,4 +115,4 @@ def complete_task():
 
   # get page to update when this method finishes
   tasks()
-  return render_template("status.html")
\ No newline at end of file
+  return key
```

Comment:
This change looks like it doesn't do what it used to do before.
Is that okay?

Reply:
I changed it so that it no longer returns render_template as I realized render_template wasn't actually being used anyway. I think it should work fine now!